easybuild/easyblocks/g/geant4.py

```diff
@@ -83,6 +96,49 @@ def configure_step(self):
 
         # Geant4 switched to a CMake build system in version 9.4
         if LooseVersion(self.version) >= LooseVersion("9.4"):
+            # map toolssg backends:
+            toolssg_backends = {'X11': 'X11', 'XT': 'Motif', 'QT': 'Qt5'}
+            if self.cfg['enable_toolssg']:
+                toolssg = toolssg_backends[self.cfg['enable_toolssg']]
+            else:
+                toolssg = []
+
+            xerces_root = get_software_root('Xerces-C++')
+
+            # map software requirements and corresponding configopts to custom options
+            custom_options_map = {
+                # option: ([dep_names], configopt)
+                'enable_freetype': (['freetype'], "-DGEANT4_USE_FREETYPE=ON"),
```

Comment:
We would normally do this the other way around.
I.e., if the easyconfig has freetype as a dependency, turn on GEANT4_USE_FREETYPE
That makes for less things to tinker with in the easyconfig.
The same of course goes for everything else with simple dep(s) -> enable-something